corehq/apps/userreports/tests/test_export.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+from django.test import SimpleTestCase
+
+from .test_data_source_config import get_sample_data_source
+from ..sql import get_indicator_table
+from ..views import process_url_params
+
+
+class ParameterTest(SimpleTestCase):
```

Comment:
This is probably the easiest place to see what the parameters look like.
`?date-lastndays=30` - filter the `date` field by the last 30 days
`?count-range=10..30` - filter the `count` field for values between `10` and `30`